action.yml

```diff
@@ -116,6 +126,7 @@ runs:
     - if: ${{ always() }}
       run: |
         rm -r "${{ inputs.name }}"
-        rm "${{ steps.dist.outputs.archive }}"
+        mkdir -p ~/.cache/
+        mv "${{ steps.dist.outputs.archive }}" ~/.cache/
```

Comment:
@galargh  I was unsure where to put it, but knew we want to keep `${{ inputs.working-directory }}`  intact, so decided to use `~/.cache` – feel free to change if there is a better place.

Reply:
Actually, I was thinking of ditching the `working-directory` altogether and using `runner.temp` directory instead for storing the intermediate artifacts. This should simplify the action after this change. I'll take care of it - likely tomorrow.